clients/vscode-hlasmplugin/.vscodeignore

```diff
@@ -8,7 +8,7 @@ tsconfig.json
 CMakeLists.txt
 hlasmplugin.log
 build/bin/**
-lib/test/**
+lib/**
```

Comment:
alternatively one could remove everything and then explicitly only add what is needed like in https://github.com/eclipse-che4z/che-che4z-explorer-for-endevor/blob/master/packages/explorer-for-endevor/.vscodeignore